dotcom-rendering/src/amp/components/Footer.tsx

```diff
@@ -6,16 +6,106 @@ import {
 	neutral,
 	textSans,
 } from '@guardian/source-foundations';
-import type { Link } from '../../lib/footer-links';
-import {
-	footerLinksNew,
-	isOnPlatform,
-	LinkPlatform,
-} from '../../lib/footer-links';
 import type { NavType } from '../../model/extract-nav';
 import { useContentABTestGroup } from './ContentABTest';
 import { ReaderRevenueButton } from './ReaderRevenueButton';
 
+export enum LinkPlatform {
+	Web,
+	Amp,
+}
+
+export interface Link {
+	title: string;
+	url?: string;
+	on?: string;
```

Comment:
I guess this property and the `LinkPlatform` enum are no longer needed as this is only used for Amp

Reply:
I've removed `LinkPlatform` - good catch!

`on` in this case refers to [something analogous to `onClick` for amp](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/amp/components/Footer.tsx#L150-L152).